internal/broker/converged_region_provider.go

```diff
@@ -39,9 +49,24 @@ func (c *DefaultConvergedCloudRegionsProvider) GetRegions(mappedRegion string) [
 	return item
 }
 
+func NewOneForAllConvergedCloudRegionsProvider() ConvergedCloudRegionProvider {
+	return &OneForAllConvergedCloudRegionsProvider{}
+}
+
 type OneForAllConvergedCloudRegionsProvider struct {
 }
 
 func (c *OneForAllConvergedCloudRegionsProvider) GetRegions(mappedRegion string) []string {
```

Comment:
Should we still return single hard-wired region?

Reply:
Just for all KEB tests that does not regard this functionality. I focused on testing current behaviour but other tests do not "see" region mapping functionality.